openstack/networking/v2/extensions/layer3/portforwarding/results.go

```diff
@@ -0,0 +1,51 @@
+package portforwarding
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+type PortForwarding struct {
+	// The ID of the floating IP port forwarding
+	ID string `json:"id"`
+
+	// The ID of the Neutron port associated to the floating IP port forwarding.
+	InternalPortID string `json:"internal_port_id"`
+
+	// The TCP/UDP/other protocol port number of the port forwarding’s floating IP address.
+	ExternalPort int `json:"external_port"`
+
+	// The IP protocol used in the floating IP port forwarding.
+	Protocol string `json:"protocol"`
+
+	// The TCP/UDP/other protocol port number of the Neutron port fixed
+	// IP address associated to the floating ip port forwarding.
+	InternalPort int `json:"internal_port"`
+
+	// The fixed IPv4 address of the Neutron port associated
+	// to the floating IP port forwarding.
+	InternalIPAddress string `json:"internal_ip_address"`
+
+	// The project ID
+	ProjectID string `json:"project_id"`
```

Comment:
I'm not seeing `project_id` in either the create results or a GET result:

```
2019/09/01 00:17:47 [DEBUG] OpenStack Response Body: {
  "port_forwarding": {
    "external_port": 2230,
    "id": "ece88ca6-0d5b-4fbd-86fb-859104a1fbb6",
    "internal_ip_address": "192.168.21.6",
    "internal_port": 25,
    "internal_port_id": "6bf31ca9-b05e-44b2-8238-45a71086351e",
    "protocol": "tcp"
  }
}
```

Are you seeing it in your environment?

Reply:
No, I am not, I added it as per your comment here: https://github.com/gophercloud/gophercloud/pull/1651#discussion_r307906828 . I removed it again in the latest commit.

Reply:
This is the first I've run into Neutron not following a defined model like this. I apologize about that.